conanfile.py

```diff
@@ -0,0 +1,108 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+
+class ViamCppSdkRecipe(ConanFile):
+    name = "viam-cpp-sdk"
+    version = "0.0.10"
```

Comment:
There will need to be something in the release process to advance this number, or it will surely drift. Can we derive it somehow?

Reply:
Yeah this was bugging me as well and I stumbled on this in the old 1.x conan docs:
https://docs.conan.io/en/1.4/howtos/capture_version.html
Going to ask online if this is still considered acceptable

Reply:
If there isn't a good way, we can always doc it in the release process.